ProcessLib/LiquidFlow/LiquidFlowLocalAssembler.h

```diff
@@ -61,21 +62,23 @@ class LiquidFlowLocalAssembler : public LiquidFlowLocalAssemblerInterface
     using GlobalDimVectorType = typename ShapeMatricesType::GlobalDimVectorType;
 
 public:
-    LiquidFlowLocalAssembler(MeshLib::Element const& element,
-                             std::size_t const /*local_matrix_size*/,
-                             bool const is_axially_symmetric,
-                             unsigned const integration_order,
-                             int const gravitational_axis_id,
-                             double const gravitational_acceleration,
-                             LiquidFlowMaterialProperties& material_propertries)
+    LiquidFlowLocalAssembler(
+        MeshLib::Element const& element,
+        std::size_t const /*local_matrix_size*/,
+        bool const is_axially_symmetric,
+        unsigned const integration_order,
+        int const gravitational_axis_id,
+        double const gravitational_acceleration,
+        std::unique_ptr<LiquidFlowMaterialProperties> const&
```

Comment:
Changed. Accordingly, ``_material_properties`` in line 65 of LiquidFlowProcess.cpp is changed to ``*_material_properties``.

 See: https://github.com/ufz/ogs/pull/1631/commits/76dd96bcdf43d60b79d3f313f612d6a8c6f52f15#diff-97a96abf0d3aa134d63d918100ee67b7R65